docs/contrib/rust-and-c++.md

```diff
@@ -0,0 +1,29 @@
+# Rust and C++
+
+Taskwarrior has historically been a C++ project, but as part of an [ongoing effort to replace its storage backend](https://github.com/GothenburgBitFactory/taskwarrior/issues/2770) it now includes a Rust library called TaskChampion.
```

Comment:
Not a blocker for this PR, but IMHO this name dichotomy will be confusing long-term. I'll try to reach out to the owner of TW crate again, or perhaps we should come up with a more descriptive name if that's not possible.

Reply:
Yeah, that will be no harder later than it is now.